Products/ZenCallHome/CallHomeStatus.py

```diff
@@ -93,8 +93,11 @@ def getStatUI(self):
         """Returns status informations to UI
         """
         l = list()
-        data = dict()
-        data = pickle.loads(self.load_from_redis())
+        rdata = self.load_from_redis()
+        if rdata is None:
+            data = dict()
+        else:
+            data = pickle.loads(rdata)
```

Comment:
These lines of code are duplicated in at least two places.  Perhaps this logic should be moved into the `load_from_redis` method so that we don't have to worry about whether `None` is returned in multiple places.